### ISSUE TITLE
Developer's Guide

### ISSUE THREAD

**Issue by jonashaag** [jonashaag/klaus#47]
- structure: files, views
- terminology

**Comment by arpit1997** [comment]
Yeah right it is very hard to find a developer guide or terminology of this project. adding a developer guide would have a nice thing since it is good project and it got up to 400 github stars. so i guess it would be a good thing 

>structure: files, views
terminology

Hope to hear
:smile_cat: :smile_cat:
